druid-shell/src/lib.rs

```diff
@@ -41,6 +42,7 @@ mod platform;
 mod scale;
 mod window;
 
+pub use alert::{AlertButton, AlertIcon, AlertOptions, AlertResponse, AlertToken};
```

Comment:
Our `crate` module is getting quite convoluted, maybe we should stop reexporting everything?

Reply:
It's definitely a worthwhile question. For now I'm just following what we've been doing.

If we decide to organize them differently then that would also mean a ton of broken docs links though. So I think even if we want to change that and move things into modules, we should wait until better doc linking is stabilized which [looks like](https://github.com/rust-lang/rust/issues/43466) it might happen this year.